mbofui/app/js/controllers.js

```diff
@@ -1,16 +1,25 @@
 /*jshint strict:false */
 /* global angular, ocellus, $, L,  resolveIcon, moment, validate, _, popupLink, checkTimeSlice */
 
-ocellus.controller('mapController', ['$compile', '$scope', '$rootScope','$filter', '$timeout', '$log', 'leafletData', 'Bof' , function($compile, $scope, $rootScope, $filter, $timeout, $log, leafletData, Bof) {
+ocellus.controller('mapController', ['$compile', '$scope', '$rootScope','$filter', '$timeout', '$log', '$location', '$window', 'leafletData', 'Bof' , function($compile, $scope, $rootScope, $filter, $timeout, $log, $location, $window, leafletData, Bof) {
   // setting up init values
   // get user
   Bof.GetUser('/api/me/').then(function(userResult) {
     $rootScope.user = userResult;
     // only function inmediately invoked - get current events on page load,
     // it fires after we have a user so that we can determine
     // which belong to the current user
-    getEvents('/api/events/current/');
-    $rootScope.currentViewUrl = '/api/events/current/';
+
+    $rootScope.currentView = window.location.search.split('=')[1];
```

Comment:
Is there a more elegant way to get this field? Is it ever possible that we'll get more than one '=' in the string that's being split on?